[ERROR] Unable to connect to Orion StartShoot G3: Operation not permitted

**Describe the bug**
After manually installing the driver (tried in both Stellarmate and Astroberry servers with Kstars and Ekos) I get an error when trying to connect (inside Ekos) and it reports this error: [ERROR] Unable to connect to Orion StartShoot G3: Operation not permitted.  In Astroberry it actually seems to crash and restart but never gets past that part and just keeps crasing.  I have sent logs over to the Stellarmate team and after several days of troubleshooting (validating USB is working, camera is working, software is up to date, etc) we are stuck at this issue.   I will note that it seems to work just fine on my MAC - if I run Kstars and Ekos it will connect just fine to my Orion CCD camera. 

**To Reproduce**
Exact steps to reproduce the behavior.
1. sudo apt-get update && sudo apt-get -y install indi-orion-ssg3
2. Start Kstars and Ekos - add the camera as an item, start profile, connect to camera and fail
3. See error attached

**Expected behavior**
I expect it to continue past the connection and then show the display as normal and connection as good.

**Screenshots**
See attached logs

**Desktop (please complete the following information):**
 - OS:  Raspberry PI - Stellarmate - latest version
 - Version [e.g. 1.7.6, GIT..etc] - Stellarmate 1.7.0

**Log Files**
Make sure to enable logging and include log files - See attached
[log_18-44-36.txt](https://github.com/indilib/indi-3rdparty/files/8370833/log_18-44-36.txt)
[log_18-40-13.txt](https://github.com/indilib/indi-3rdparty/files/8370836/log_18-40-13.txt)
[HARDWARE_Details.pdf](https://github.com/indilib/indi-3rdparty/files/8370837/HARDWARE_Details.pdf)
[USB_Details.pdf](https://github.com/indilib/indi-3rdparty/files/8370838/USB_Details.pdf)
[Error.pdf](https://github.com/indilib/indi-3rdparty/files/8370878/Error.pdf)

--- a/debian/indi-orion-ssg3/indi-orion-ssg3.install
+++ b/debian/indi-orion-ssg3/indi-orion-ssg3.install
@@ -1,2 +1,3 @@
 usr/bin/
 usr/share/indi/
+lib/udev/rules.d/

